polaris.shopify.com/content/tools/stylelint-polaris/index.md

```diff
@@ -90,24 +90,24 @@ There are over 40 rules configured in Stylelint Polaris to help you avoid errors
 - [Media queries](/tools/stylelint-polaris/rules#media-queries)
 - [Motion](/tools/stylelint-polaris/rules#motion)
 - [Shape](/tools/stylelint-polaris/rules#shape)
-- [Spacing](/tools/stylelint-polaris/rules#spacing)
+- [Space](/tools/stylelint-polaris/rules#space)
```

Comment:
I'd prefer to handle this change in https://github.com/Shopify/polaris/pull/9006